### PR TITLE
Edits to Catalyst Mac Idiom Post

### PR DESCRIPTION
**Pull request by deutschbitte**

### CODE REVIEW

File: _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md [hunk 1]
```diff
@@ -55,14 +55,13 @@ if !UIDevice.current.isCatalystMacIdiom {
 }
 ```
 
-
 ## AppKit In UIKit
 
-Internally Apple uses a private `_UINSView` class to host an actual `NSView`. It's too bad we didn't get this class public, which would allow us to freely mix AppKit with UIKit, but it's a start.
+Internally, Apple uses a private `_UINSView` class to host an actual `NSView`. It’s too bad we didn’t get this class public, which would’ve allowed us to freely mix AppKit with UIKit, but it’s a start.
```
Comment (deutschbitte) [inline _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md:60]: Can we instead say "we didn't get this class publically" or maybe even "Apple didn't make this class public" ?

"It's too bad we didn't get this class public"

File: _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md [hunk 2]
```diff
@@ -111,46 +110,44 @@ We find references to following controls:
...
+However, that’s the end of the story for now. The data is there, but the tools can’t [yet (!)](https://twitter.com/bsr43/status/1308462962680659971?s=21) get a useful format out. We know that there are at least five controls that throw an exception on *some* usage at runtime, however, which one it is exactly is currently hard to know. Shipping a Catalyst app in the new Mac idiom is definitely an adventure.
 
-## Decompile via lldb
+## Decompile via LLDB
 
-Jeff Johnson points out that one can [abuse lldb to piecemeal decompile methods](https://lapcatsoftware.com/articles/bigsur3.html). That approach would take far too long to find all calls that throw here, but it's a start.
+Jeff Johnson points out that one can [abuse LLDB to piecemeal decompile methods](https://lapcatsoftware.com/articles/bigsur3.html). That approach would take far too long to find all the calls that throw here, but it’s a start.
```
Comment (deutschbitte) [inline _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md:139]: Piecemeal is more an adverb or adjective so it's use as a verb here doesn't work. Can you suggest a rephrase?

"Jeff Johnson points out that one can [abuse LLDB to piecemeal decompile methods](https://lapcatsoftware.com/articles/bigsur3.html)."

File: _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md (hunk 1 above)
Comment (steipete) [reply _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md:60]: It’s too bad Apple didn't consider making this class public API, which...

File: _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md (hunk 2 above)
Comment (steipete) [reply _posts/2020-09-22-forbidden-controls-in-catalyst-mac-idiom.md:139]: Jeff Johnson points out that one can abuse LLDB to decompile methods individually."

**Pull request merged**